components/BottomNavigation/tests/snapshot/MDCBottomNavigationBarSnapshotTests.m

```diff
@@ -139,6 +142,11 @@ - (void)changeToRTLAndArabicWithTitle:(NSString *)title {
       view.semanticContentAttribute = UISemanticContentAttributeForceRightToLeft;
     }
   }
+  if (self.navigationBar.items.count >= 2U) {
```

Comment:
I wouldn't consider this a particularly magic number, it's just a bounds check.  Item 1 is where most examples have the ink touch and I wanted to prefer a different item when possible.  Since the next number after 1 is 2, I went with that.